pkg/engine/states/local/backend.go

```diff
@@ -0,0 +1,35 @@
+package local
+
+import (
+	"github.com/zclconf/go-cty/cty"
+	"kusionstack.io/kusion/pkg/engine/states"
+)
+
+type LocalBackend struct {
+	FileSystemState
+}
+
+func NewLocalBackend() states.Backend {
```

Comment:
We can construct one backend with a params map and config/validate this backend in this method.
Ref: https://github.com/KusionStack/kusion/blob/fb656aa3adc3d79bf32497887f2cf61be296099b/pkg/engine/states/remote/http_state.go#L48

Reply:
invoker just need to define ConfigSchema、Configure interface，BackendFromConfig function handler backend config/validate